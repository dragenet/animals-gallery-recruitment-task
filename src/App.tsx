import { RouterProvider } from 'react-router-dom';

import { RootProvider } from '~/components/atoms/RootProvider';

import { browserRouter } from './router';

function App() {
  return (
    <RootProvider>
      <RouterProvider router={browserRouter} />
    </RootProvider>
  );
}

export default App;
