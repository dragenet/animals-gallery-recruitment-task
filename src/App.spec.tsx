import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import App from './App';

describe('Test app', async () => {
  it('snapshots', () => {
    const { container } = render(<App />);

    expect(container).toMatchSnapshot();
  });
});
