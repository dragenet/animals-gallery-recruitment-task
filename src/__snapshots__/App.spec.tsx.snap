// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Test app > snapshots 1`] = `
<div>
  <div
    class="sc-beqWaB fQzpRu"
  />
</div>
`;
